packages/location-field/src/index.tsx

```diff
@@ -201,132 +202,127 @@ const LocationField = ({
     }
   }, [initialSearchResults]);
 
-  const geocodeAutocomplete = useMemo(
-    () =>
-      debounce(300, (text: string) => {
-        if (!text) {
-          console.warn(
-            "No text entry provided for geocode autocomplete search."
-          );
-          setMessage(null);
-          return;
-        }
-        setFetching(true);
-        setMessage(
-          intl.formatMessage({
-            defaultMessage: "Fetching suggestions…",
-            description:
-              "Hint shown while geocoder suggestions are being fetched",
-            id: "otpUi.LocationField.fetchingSuggestions"
-          })
-        );
+  const geocodeAutocomplete = debounce(300, (text: string) => {
```

Comment:
Would have to check in the future if we should consider a `useCallback`.

Reply:
I've added a comment